tailwind.config.js

```diff
@@ -1,29 +1,25 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-  content: [
-    "./src/**/*.{js,jsx,ts,tsx}",
-  ],
+  content: ['./src/**/*.{js,jsx,ts,tsx}', './src/assets/*.json'],
```

Comment:
So, your Tailwind CSS classes were not generated, because Tailwind only generates classes that it can find the files specified here. Because of Tailwinds Just In Time compliation, it doesn't actually support fully dynamic classes, only the ones that it can find.

https://tailwindcss.com/docs/content-configuration#dynamic-class-names

Also, I generally don't recommend using this approach of putting Tailwind class names as props. There are 2 possible, and more ideal solutions for this:

1. Specify a given amount of possible classes or colors, and then map them inside the component
2. If really all colors on the color spectrum should be possible, then just simply specify the hex and add it as inline styles